docs/04-reference/wingsdk-api.md

```diff
@@ -1557,39 +1557,39 @@ The request's query string values.
 
 ---
 
-##### `body`<sup>Optional</sup> <a name="body" id="@winglang/sdk.cloud.ApiRequest.property.body"></a>
+##### `vars`<sup>Required</sup> <a name="vars" id="@winglang/sdk.cloud.ApiRequest.property.vars"></a>
```

Comment:
Hey @garysassano we would be grateful if you could also update the relevant part of [Wing SDK Spec](https://github.com/winglang/wing/blob/main/docs/04-reference/wingsdk-spec.md#api) as part of your PR 🙏

Reply:
Thanks a lot for your input, updated that as well.